twphotos/photos.py

```diff
@@ -85,23 +87,34 @@ def load(self, user=None, count=None, max_id=None,
              since_id=None, num=None, photos=None):
         if photos is None:
             photos = []
-        statuses = self.api.GetUserTimeline(
-            screen_name=user,
-            count=count or COUNT_PER_GET,
-            max_id=max_id,
-            since_id=since_id,
-            exclude_replies=self.exclude_replies)
+
+        if self.tl_type == 'favorites':
+            statuses = self.api.GetFavorites(
+                screen_name=user,
+                count=count or COUNT_PER_GET,
+                max_id=max_id,
+                since_id=since_id)
+        else:
+            statuses = self.api.GetUserTimeline(
+                screen_name=user,
+                count=count or COUNT_PER_GET,
+                max_id=max_id,
+                since_id=since_id,
+                exclude_replies=self.exclude_replies)
+
         if statuses:
             min_id = statuses[-1].id
             max_id = statuses[0].id
             self.max_ids.setdefault(user, max_id)
 
         fetched_photos = []
         for s in statuses:
-            for m in s.media:
-                if m['type'] == 'photo':
-                    t = (m['id'], m['media_url'])
-                    fetched_photos.append(t)
+            if s.media is not None:
+                for m in s.media:
+                    m_dict = m.AsDict()
```

Comment:
Perhaps `TwitterModel` specification has changed.
When `Media` is refered as a Dictionaly, a error occurred.
https://python-twitter.readthedocs.io/en/latest/twitter.html#twitter.models.TwitterModel